services/app-web/src/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,57 @@
+import { useState, useEffect } from 'react'
+import { LocalStorageKeyType } from '../constants/localStorage'
+type LocalStorage = {
+    key: LocalStorageKeyType
+    value: string | null
+}
+
+type UseLocalStorage = [LocalStorage['value'], (value: string | null) => void]
+// Get and set keys in local storage. If key is set to a value of null, clear and remove from local storage, return default fallback value
+function useLocalStorage(
+    key: LocalStorage['key'],
+    defaultValue: LocalStorage['value']
+): UseLocalStorage {
+    const [storedValue, setStoredValue] = useState<string | null>(() => {
+        if (typeof window === 'undefined') {
+            console.error('Unable to find local storage. window is undefined')
+            return defaultValue
+        }
+
+        let value
+        try {
+            value = JSON.parse(
```

Comment:
since values can only be strings or null, why are we JSON parsing / stringifying this?